pkg/advancedapi/advancedApi_test.go

```diff
@@ -409,12 +409,17 @@ func Test_can_set_document_controller(t *testing.T) {
 	twinId, err := advancedapi.NewRegisteredIdentity(resolver, identity.Twin, test.ValidKeyPairPlop, "#ExistingId", false)
 	assert.NilError(t, err)
 
+	initialDoc, err := resolver.GetDocument(twinId.Did())
+	assert.NilError(t, err)
+
+	time.Sleep(time.Millisecond) // Note: Need to sleep to ensure UpdateTime is updated on document build
```

Comment:
:+1: 
(unimportant) Worth noting here that update time granularity is in Milliseconds? (I assume)

Reply:
Yes, DID spec says update time should be milliseconds.